examples/cms-prepr/components/date.js

```diff
@@ -1,6 +1,13 @@
 import { parseISO, format } from 'date-fns'
+import { useEffect, useState } from 'react'
 
-export default function Date({ dateString }) {
-  const date = parseISO(dateString)
-  return <time dateTime={dateString}>{format(date, 'LLLL	d, yyyy')}</time>
+// dateString might be null for unpublished posts
```

Comment:
If null, let's just not render the component, versus needing `useEffect`.

Reply:
Changed in newest commit